src/u2f/u2f_app.c

```diff
@@ -7,15 +7,42 @@
 
 #include <stddef.h>
 #include <stdio.h>
+#include <wally_bip39.h>
 
 #define APPID_BOGUS_CHROMIUM "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
 #define APPID_BOGUS_FIREFOX "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
 
+#define SHORT_MNEMONIC_NUM_WORDS 4
+
 typedef struct {
     uint8_t app_id[32];
     const char* name;
 } app_t;
 
+// shorten (and split as array) menonic for displaying hashes in a more recognizable way, at the
+// cost of less entropy, output configurable for other use return success
+static bool _short_mnemonic(char* mnemonic, const char** wordlist_out, uint8_t num_words)
+{
+    // use zero for default
+    if (num_words == 0) {
+        num_words = SHORT_MNEMONIC_NUM_WORDS;
+    }
```

Comment:
please delete, this is dead code, along with the then unused `SHORT_MNEMONIC_NUM_WORDS` definition.

Reply:
to elaborate, the definition does not help as you can't change it to anything else and expect it to work. so the magic `4` at the call site is good in this exceptional case ;)

Reply:
okay...